packages/babel-preset-react-app/loader.js

```diff
@@ -6,7 +6,28 @@
  */
 'use strict';
 
-const loader = require('babel-loader');
-const overrides = require('./overrides');
+const crypto = require('crypto');
 
-module.exports = loader.custom(() => overrides);
+module.exports = function() {
```

Comment:
Let's name this file something else, e.g. `overrides` or `loader-customization`.

Reply:
Maybe `babel-preset-react-app/webpack-overrides`

Reply:
`babel-overrides`?

Reply:
Internally, Babel uses `overrides` so I like that -- this is specific to the webpack loader though.

`webpack-overrides` is good, but `loader-overrides` would work too.